website/content/en/highlights/2023-11-07-0-34-0-upgrade-guide.md

```diff
@@ -0,0 +1,26 @@
+---
+date: "2023-11-07"
+title: "0.34 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.34.0"
+authors: ["dsmith3197"]
+release: "0.34.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+Vector's 0.34.0 release includes **breaking changes**:
+
+1. [Removal of Deprecated Datadog Component Config Options](#datadog-deprecated-config-options)
+
+We cover them below to help you upgrade quickly:
+
+## Upgrade guide
+
+### Breaking changes
+
+#### Removal of Deprecated Datadog Component Config Options {#datadog-deprecated-config-options}
+
+The `region` config option has been removed from the Datadog Events, Logs,
+and Metrics sinks. The `region` and `application_key` config options have
+been removed from the Enterprise configuration.
```

Comment:
```suggestion
The `region` config option has been removed from the Datadog Events, Logs,
and Metrics sinks. Instead the `site` option should be used.

The `region` and `application_key` config options have
been removed from the Enterprise configuration. Instead of `region`, `site` should be used. `application_key` is no longer required.
```